advanced/editor-command-identifiers.md

```diff
@@ -460,6 +462,7 @@ The following command states can be queried using the [queryCommandState]({{ sit
 | Strikethrough        | Returns `true` if the content is formatted using the same markup as the {{site.productname}} `Strikethrough` command.          |
 | Subscript            | Returns `true` if the content is formatted using the same markup as the {{site.productname}} `Subscript` command.              |
 | Superscript          | Returns `true` if the content is formatted using the same markup as the {{site.productname}} `Superscript` command.            |
+| ToggleToolbarDrawer  | Returns `true` if the toolbar drawer is open. The state can be controlled by the {{site.productname}} `ToggleToolbarDrawer` command. |
```

Comment:
Likewise, does this need the `{{ site.requires_5_5v }}` note?

Reply:
Fixed